astropy/units/quantity.py

```diff
@@ -11,13 +11,15 @@
 import re
 import numbers
 from fractions import Fraction
+import operator
 import warnings
 
 import numpy as np
 
 # AstroPy
 from .core import (Unit, dimensionless_unscaled, get_current_unit_registry,
                    UnitBase, UnitsError, UnitConversionError, UnitTypeError)
+from .structured import StructuredUnit
```

Comment:
How is structured units going to work with SpecificTypeQuantities? I don't see any tests for this case. Does SpecificTypeQuantity properly work on each field?

Reply:
As is, for the ones that we have, the units would not be equivalent, so it wouldn't work.

But you're right that it would be a nice test case to have a `PositionVelocity` that holds an erfa `pv` unit!